src/DataTrailCard.tsx

```diff
@@ -58,9 +60,11 @@ export function DataTrailCard(props: Props) {
   const { filters, metric, createdAt } = values;
 
   const heading = truncateValue('', getMetricName(metric), 39);
```

Comment:
per @yangkb09's commen about metric name truncation, this looks correct when set to `27`:

![truncate set to 27](https://github.com/user-attachments/assets/8f2f9613-6f13-4a64-aaaf-181a947f5e2d)

```suggestion
  const heading = truncateValue('', getMetricName(metric), 27);
```